### PR TITLE
Added Keylime attestation

### PR DESCRIPTION
**Pull request by tbazant**
### Description

Keylime is the remote attestation implementation in ALP.

### Are there any relevant issues/feature requests?

* jsc#PED-3624

### CODE REVIEW

File: concepts/keylime.xml [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+            and has not been tampered with.
+          </para>
+        </listitem>
+      </varlistentry>
+      <varlistentry>
+        <term>Core root of trust for measurement</term>
+        <listitem>
+          <para>
+            A starting point of the boot process that cannot be altered and
+            calculates the first hash of the layer above.
```
Comment (aplanas) [inline concepts/keylime.xml:55]: Calculates its how hash, and the hash of the next step in the boot process, initiating the chain of measurements

File: concepts/keylime.xml [hunk 2]
```diff
@@ -0,0 +1,238 @@
...
+          </para>
+        </listitem>
+      </varlistentry>
+      <varlistentry>
+        <term>Endorsement key (EK)</term>
+        <listitem>
+          <para>
+            An encryption key that is permanently embedded in the TPM when it
+            is manufactured. The public part of the key is used to recognize a
+            genuine TPM.
```
Comment (aplanas) [inline concepts/keylime.xml:65]: " The public part of the key and an certification stored in the TPM are used to recognize a genuine TPM."

File: tasks/keylime-setup-verifier-and-registrar.xml [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+        the health of remote nodes. The <emphasis>verifier</emphasis> and
+        <emphasis>registrar</emphasis> are essential components of &keylime; on
+        remote systems to perform the registration and attestation of &keylime;
+        agents.
+      </para>
+    </abstract>
+  </info>
+  <para>
+    Before you start installing and registering agents, prepare the verifier
+    and the registrar on remote hosts as described in the following procedure.
```
Comment (aplanas) [inline tasks/keylime-setup-verifier-and-registrar.xml:33]: Over the time we make a container that has the verifier, registrar and tenant services. There are instructions here about how to install and use it:

https://build.opensuse.org/package/view_file/devel:microos:containers/keylime-control-plane-image/README?expand=1

File: concepts/keylime.xml (hunk 1 above)
Comment (tbazant) [reply concepts/keylime.xml:55]: > Calculates its how hash, 
i don't understand this, can you fix/make clear pls?

File: tasks/keylime-setup-verifier-and-registrar.xml (hunk 3 above)
Comment (tbazant) [reply tasks/keylime-setup-verifier-and-registrar.xml:33]: @lvicoun i'm going to rewrite this task, verify if it applies to SLE Micro as well pls

**Review by tbazant — COMMENTED** [review]
thanks for the review!

File: concepts/keylime.xml (hunk 1 above)
Comment (aplanas) [reply concepts/keylime.xml:55]: Calculates its ~~how~~ own hash, ... (sorry for the typo)

Comment (tbazant) [inline concepts/keylime.xml:55]:
```suggestion
            Calculates its own hash and the hash of the next step in the boot process, initiating the chain of measurements.
```

**Review by aplanas — APPROVED** [review]
Thanks for adding the container part!

**Pull request merged**